src/pages/resources/recipes/html-events/index.md

```diff
@@ -0,0 +1,86 @@
+
+# HTML events and event listeners
+On the web, you can use different techniques to capture an HTML event and add `eventlisteners`` to an element. UXP too, supports these ways. 
+
+It is simply your preference; the method you feel most comfortable with. However, there might be a need for you to configure the right permission for inline event handlers, such as `<button onclick="btnClick()">Click Me</button>` 
+
+
+<InlineAlert variant="info" slots="header, text1, text2"/>
+
+Scripts and plugins
+
+**In plugins**, you will need to enable `allowCodeGenerationFromStrings` if you would like to use the inline event handlers shown above. <br></br>
```

Comment:
@kareemmyk  Is allowCodeGenerationFromStrings used for anything else?

Reply:
@kareemmyk  Not sure I understand Sourcemaps using "eval"? Can you please elaborate? I can add it o the manifest tutorial.

Reply:
Sourcemaps using eval is generally needed for Webpacked files and is usually used while debugging.

We can include only ```eval and new functions()``` for now.

Reply:
@kareemmyk  please make a note about this while working on the webpack starter template.